packages/fast-components/src/badge/fixtures/base.html

```diff
@@ -0,0 +1,71 @@
+<fast-design-system-provider style="font-family: var(--body-font)">
+    <style>
+        fast-badge {
+            --badge-fill-primary: #D223CB;
+            --badge-fill-secondary: #3278CD;
+            --badge-fill-tertiary: #DFDFDF;
+            --badge-fill-foo: #EE1010;
+            --badge-fill-bar: rgba(238,6,16,0.7);
+            --badge-color-foo: white;
+            --badge-color-bar: black;
+        }
+    </style>
+    <h1>Badge</h1>
+    <h4>Default</h4>
+    <fast-badge fill="primary" style="font-size: 10px; line-height: 16px;" color="foo" id="foo" aria-label="Simple badge" modal="true">
```

Comment:
I'm not sure I'm fully understanding the purpose of setting all the font properties as inline styles here.

Reply:
Just to show that its possible and that the badge properly handles it